data/terms.json

```diff
@@ -729,8 +728,8 @@
     "meaning": "The 'read-write web,' which began in the 1990s, is distinguished by user-generated content and enhanced user interfaces. Web2 emphasizes user experience and interoperability across different applications and websites, resulting in an extensive network of linked websites."
   },
   {
-    "term": "web2.0",
-    "meaning": "The 'read-write web,' which began in the 1990s, is distinguished by user-generated content and enhanced user interfaces. Web2 emphasizes user experience and interoperability across different applications and websites, resulting in an extensive network of linked websites."
+    "term": "web2",
+    "meaning": "I love the JavaScript ecosystem as it is used both in the web 2.0 as well as web 3.0. If you are a javascript developer, stick to the language and get better at it. It's flexibilty and extensibility across the board cannot be overemphasized."
```

Comment:
This definition seemed to be more of a joke

Reply:
> This definition seemed to be more of a joke

Exactly, not sure what he is trying to say @MajorReact

Reply:
> > This definition seemed to be more of a joke
> 
> Exactly, not sure what he is trying to say @MajorReact

Sorry about the weird headline. I failed some tests while trying to make some contributions and earn some badges at the #hacktoberfest. The prebuilt tests already set, help me understand testing more. It's my first-time using jest and I gained some experience while making contributions. My name is Nnamdi from Nigeria by the way. I follow you on twitter and I saw you shared the repo for others to contribute.

Reply:
> > This definition seemed to be more of a joke
> 
> Exactly, not sure what he is trying to say @MajorReact

Will it be accepted?